.github/ISSUE_TEMPLATE/---report-a-bug.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "\U0001F41B Report a bug"
+about: Create a report to help us fix bugs
+title: ''
+labels: bug
+assignees: drahnr
+
+---
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
```

Comment:
Do we want to add in some examples of bug descriptions? Sometimes people aren't aware of how long/short a description we need to resolve something.

i.e. 
"Juice failed to build with cargo commands from README"
"Network fails to converge using Juice RNN"